x/model/client/cli/query_model_test.go

```diff
@@ -53,47 +51,43 @@ func TestShowModel(t *testing.T) {
 		idVid int32
 		idPid int32
 
-		args []string
-		err  error
-		obj  types.Model
+		common []string
```

Comment:
Can we add negative tests for missing or invalid parameters (flags)?

Reply:
Added TODO comments about necessity to add such negative tests.